src/toolbox.ts

```diff
@@ -1091,24 +1088,29 @@ export class AnnotationResizeItem extends ToolboxItem {
             switch(event.key) {
                 case this.keybind_configuration.annotation_vanish.toUpperCase():
                     this.update_all_subtask_annotation_size(this.ulabel, "v");
```

Comment:
It's unrelated to this PR, but shouldn't these function calls pass `event.key` instead of a constant?

Reply:
The resizing functions inside this switch statement expects constants "s", "m", "l", "-", or "=". So the switch statement just compares the currently pressed key against the value in the config, then calls the resize function with the associated string constant. 

instead of having the function it calls compare against the values in the config, I believe its done this way because the resizing buttons call the resizing function with predetermined values.

Reply:
Should use an `Enum` or `const` then

Reply:
Done in [c47de5b](https://github.com/SenteraLLC/ulabel/pull/121/commits/c47de5b777d6bc169e3f0a1bd49f63145e8a5fe0)